doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -1626,7 +1626,11 @@ The second entry in the :token:`i_view` grammar rule,
 ``/ltac:(`` :token:`tactic` ``)``, executes :token:`tactic`.
 Notations can be used to name tactics,  for example
 
-.. coqtop:: in fail
+.. coqtop:: none warn
```

Comment:
This one `warn` should not be necessary:
```suggestion
.. coqtop:: none
```

Reply:
Oups, forgotten there, now corrected.